Shared/Samples/Style symbols from mobile style file/StyleSymbolsFromMobileStyleFileView.swift

```diff
@@ -0,0 +1,189 @@
+            // Get the keys from the option selections.
+            var symbolKeys = ["Face1"]
+            symbolKeys.append(contentsOf: symbolOptionSelections.categoryKeys.map { $0.value })
+            
+            // Get the symbol from symbol style using the keys.
+            if let pointSymbol = try? await symbolStyle.symbol(forKeys: symbolKeys) as? MultilayerPointSymbol {
+                // Color lock all layers but the first one.
+                let layers = pointSymbol.symbolLayers
+                for (i, layer) in layers.enumerated() {
+                    layer.colorIsLocked = i != 0 ? true : false
+                }
+                
+                pointSymbol.color = UIColor(symbolOptionSelections.color)
+                pointSymbol.size = symbolOptionSelections.size
+                
+                // Create an image swatch for the symbol using the display scale.
+                if let swatch = try? await pointSymbol.makeSwatch(scale: displayScale) {
+                    // Update the current symbol with the created symbol and swatch.
+                    currentSymbol = SymbolDetails(symbol: pointSymbol, image: swatch)
+                }
+            }
+        }
+    }
+    
+    /// A symbol and its associated information.
+    struct SymbolDetails {
+        /// The symbol from the symbol style.
+        let symbol: Symbol
+        /// The image swatch of the symbol.
+        let image: UIImage
+        /// The name of the symbol as found in the symbol style.
+        let name: String
+        /// The key of the symbol as found in the symbol style.
+        let key: String
+        /// The category of the symbol as found in the symbol style.
+        let category: String
+        /// The human-readable label of the symbol name.
+        var label: String {
+            let splitName = name.replacingOccurrences(of: "-", with: " ").split(separator: " ")
+            return splitName.last?.capitalized ?? name
+        }
+        
+        init(symbol: Symbol, image: UIImage, name: String = "", key: String = "", category: String = "") {
+            self.symbol = symbol
+            self.image = image
+            self.name = name
+            self.key = key
+            self.category = category
+        }
+    }
+    
+    /// The different options used in creating a symbol from the symbol style.
+    struct SymbolOptions: Equatable {
+        /// A dictionary containing the key of a symbol for each symbol category.
+        var categoryKeys: [SymbolCategory: String] = [:]
+        /// The color of the symbol.
+        var color: Color
+        /// The size of the symbol.
+        var size: Double
+        
+        init(eyes: String, hat: String, mouth: String, color: Color, size: Double) {
+            categoryKeys[.eyes] = eyes
+            categoryKeys[.hat] = hat
+            categoryKeys[.mouth] = mouth
+            self.color = color
+            self.size = size
+        }
+    }
```

Comment:
categoryKeys does not need to be initialized with `[:]`. Also, can just be assigned once:

```swift
    struct SymbolOptions: Equatable {
        /// A dictionary containing the key of a symbol for each symbol category.
        var categoryKeys: [SymbolCategory: String]
        /// The color of the symbol.
        var color: Color
        /// The size of the symbol.
        var size: Double
        
        init(eyes: String, hat: String, mouth: String, color: Color, size: Double) {
            categoryKeys = [
                .eyes: eyes,
                .hat: hat,
                .mouth: mouth
            ]
            self.color = color
            self.size = size
        }
    }
```